ui/src/components/HarPage.tsx

```diff
@@ -1,14 +1,14 @@
 import React, {useEffect, useRef, useState} from "react";
```

Comment:
if it is easy to rename it to traffic page - I think it is a better name (because it is not HAR).
the changes was recently done to allow this multiple protocols if I'm not mistaken